src/js/components/programs.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable import/prefer-default-export */
+import { newText, newMultimedia, newContainer } from './helpers';
+
+/**
+ * @param {array[nodesHTMl]} cardsContainer es el array de nodosHTML que contiene el
+ *  ul los li y las cards
+ * crea unos divs invisibles que serviran para mover las cards del slider dando clic
+ * agrega la funcionalidad para que al darle clic a los divs invisibles de scroll tanto
+ *  a derecha como a izquierda
+ */
+const createControls = (cardsContainer) => {
```

Comment:
Juan, please remember to put an ID: 'scroll' in the main container of the component, since it is necessary for the functionality of my scroll

Reply:
done